airbyte-integrations/connectors/source-aha/source_aha/source.py

```diff
@@ -13,6 +21,49 @@
 
 
 # Declarative Source
-class SourceAha(YamlDeclarativeSource):
+class Aha(YamlDeclarativeSource):
     def __init__(self):
         super().__init__(**{"path_to_yaml": "manifest.yaml"})
+
+
+def check_existence(config: Mapping[str, Any], url_param: str) -> Union[tuple[bool, None], tuple[bool, str]]:
```

Comment:
for checking existance of stream - as I said in PR - it's returning `403` with listed roles